util.sh

```diff
@@ -30,14 +32,10 @@ function check::depot-tools() {
   if [ ! -d $depot_tools_dir ]; then
     git clone -q $depot_tools_url $depot_tools_dir
     if [ $platform = 'windows' ]; then
-      # set up task to run gclient.bat to get python
-      schtasks //Create //tn init_gclient //tr `cd $depot_tools_dir; pwd -W`/gclient.bat //sc onstart //f //RU system
-      schtasks //Run //tn init_gclient
-      sleep 1
-      schtasks //Delete //tn init_gclient //f
-      while [ ! -f $depot_tools_dir/python276_bin/python.exe ]; do
-        sleep 5
-      done
+      # run gclient.bat to get python
+      pushd $depot_tools_dir
+      ./gclient.bat
```

Comment:
Just a bit of context: before, when I was regularly using Windows, I recall ./gclient.bat wouldn't work as expected when run from the msys shell; it would appear to run but dependencies that it pulled in like python wouldn't exist. When I ran gclient.bat from the windows prompt or executed it manually from Explorer, the dependencies were pulled in successfully. 

`schtasks` was a kludge to execute gclient in such a way that gclient pulled in dependencies successfully. This behavior must have changed since then.

I'm between Windows environments at the moment, so this looks good to me AFAICT.